kieserver/image.yaml

```diff
@@ -199,7 +199,31 @@ envs:
     description: "Disable Maven pull dependencies for immutable KIE Server configurations for S2I and pre built kjars. Useful for pre-compiled kjar."
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
-    description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+    description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."  
+  - name: "KIE_KAFKA_EXT_DISABLED"
+    example: "true"
+    description: "Disable Kafka extension"
```

Comment:
Improve the description with more details and also which system property it sets.
e.g.:
https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/524/files#diff-9c3271200df1057c7c0e0dd7542d17d0e2467adf63351fbb5cd6d51efe9b6062R140

Reply:
same for the others.

Reply:
And, when it does have a default value, specify it as well.

Reply:
ok